metron-analytics/metron-profiler-client/src/main/java/org/apache/metron/profiler/client/stellar/GetProfile.java

```diff
@@ -82,19 +84,53 @@
 public class GetProfile implements StellarFunction {
 
   /**
-   * A global property that defines the name of the HBase table storing profile definitions.
+   * A global property that defines the name of the HBase table used to store profile data.
    */
-  public static final String PROFILER_HBASE_TABLE = "profiler.hbase.table";
+  public static final String PROFILER_HBASE_TABLE = "profiler.client.hbase.table";
 
   /**
    * A global property that defines the name of the column family used to store profile data.
    */
-  public static final String PROFILER_COLUMN_FAMILY = "profiler.column.family";
+  public static final String PROFILER_COLUMN_FAMILY = "profiler.client.hbase.column.family";
 
   /**
    * A global property that defines the name of the HBaseTableProvider implementation class.
    */
-  public static final String PROFILER_HBASE_TABLE_PROVIDER = "profiler.hbase.table.provider";
+  public static final String PROFILER_HBASE_TABLE_PROVIDER = "hbase.provider.impl";
```

Comment:
For example, EnrichmentIntegrationTest.java:98 or ProfilerIntegrationTest.java:266.  Open to suggestions